src/main/kotlin/com/cognifide/gradle/aem/environment/docker/container/HostFileManager.kt

```diff
@@ -0,0 +1,51 @@
+package com.cognifide.gradle.aem.environment.docker.container
+
+import com.cognifide.gradle.aem.common.file.resolver.FileResolver
+import com.cognifide.gradle.aem.environment.docker.Container
+import org.gradle.util.GFileUtils
+import java.io.File
+
+/**
+ * File manager for host OS files related specific Docker container.
+ * Provides DSL for e.g creating directories for volumes and providing extra files shared via volumes.
+ */
+class HostFileManager(val container: Container) {
+
+    private val aem = container.aem
+
+    private val logger = aem.logger
+
+    private val docker = container.docker
+
+    val rootDir = File(docker.environment.rootDir, container.name)
+
+    var fileDir = File(rootDir, aem.props.string("environment.container.host.fileDir") ?: "files")
+
+    fun file(path: String) = File(rootDir, path)
+
+    val configDir = File(docker.environment.configDir, container.name)
+
+    fun configFile(path: String) = File(configDir, path)
+
+    fun resolveFiles(options: FileResolver.() -> Unit): List<File> {
```

Comment:
`resolve { resolveFiles {} }` looks little redundant, maybe `resolve { files {}}` only?

Reply:
hmmm also confusing...

![image](https://user-images.githubusercontent.com/10673561/66907328-7207da80-f009-11e9-88ee-cffd052ecc25.png)

Reply:
`file()` is an accessor, which intentionally hides `Project.file()`

`files()` in Gradle is usually a factory method for `Project.files() : FileCollection`. I'd like to avoid such overlapping here because ... this method is doing completely different things

Reply:
OK

Reply:
OK